### DISCUSSION TITLE
Upgrading a database droplet.

### DISCUSSION THREAD

**Discussion by Dlewis89** [digitalocean/terraform-provider-digitalocean#872] [Q&A]
I'm attempting to upgrade my database droplet from 1vcpu, 1gbram to 1vcpu, and 2gbram but I need to make sure that I understand how that works. 

The way I believe this would be done is change the size attr from "db-s-1vcpu-1gb" to "db-s-1vcpu-2gb" and it would upgrade the droplet.

Just want to make sure that it's not going to recreate the droplet deleting all of the contents. 

Thank you for an help or advice you can provide.

**Comment by andrewsomething** [comment]
Hi @Dlewis89 

When you say "database droplet " do you mean a Droplet you have installed a database on or the node size for a managed database cluster? Can you share an example config? Both support scaling up in place, but some details are different. 

`terraform plan` should show you the proposed operation, and unless you run `terraform apply` with the `-auto-approve` flag you are asked to approve any operation that would destroy the resource.

**Reply by Dlewis89** [reply]
Hi @andrewsomething 

I'm sorry, very new to terraform, so I'm kind of figuring this out as I go but I really appreciate the help.

resource "digitalocean_database_cluster" "db-mysql-nyc1-22966" {
  name                 = "db-mysql-nyc1-22966"
  engine               = "mysql"
  version              = "8"
  size                 = "db-s-1vcpu-1gb"
  region               = "nyc1"
  node_count           = 1
  private_network_uuid = digitalocean_vpc.default-nyc1.id
}

Here is the configuration for the cluster. I hope this is what you were referring too.

**Reply by andrewsomething** [reply]
Changing the size to `db-s-1vcpu-2gb` should be a safe operation here. One thing to note is that you will not be able to scale back down to a smaller size as you can not shrink the disk space.

**Reply by Dlewis89** [reply]
Awesome!!

Thank you so much for your help @andrewsomething
